tests/testthat/test-half.life.R

```diff
@@ -333,4 +333,22 @@ test_that("get_halflife_points", {
     suppressMessages(get_halflife_points(o_nca)),
     regexp = "More than one half-life calculation was attempted on the following rows: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11"
   )
+
+  # Half-life points guess right even if lambda.z.time.last != tlast
+  d_conc <- data.frame(
+    conc = c(1, 0.5, 0.25, 0.125, 0.07, 0, 0.01),
+    time = c(0, 1, 2, 3, 4, 5, 6),
+    exclude_hl = c(FALSE, FALSE, FALSE, FALSE, FALSE, FALSE, TRUE),
+    subject = 1
+  )
+  o_conc <- PKNCAconc(d_conc, formula = conc ~ time | subject, exclude_half.life = "exclude_hl")

```

Comment:
Please add one more test that does not use the `exclude_hl` column in `PKNCAconc()` to verify that the results are different. Having the positive and negative tests ensures that the feature is working as intended and not accidentally short-circuited at some point.

Reply:
Sorry for the confusion. I was meaning something simpler. I was looking for one test including `tlast` and the other test with the same data excluding it. I updated the test.